.golangci.yml

```diff
@@ -0,0 +1,88 @@
+linters:
```

Comment:
[nit] since you're enabling `misspell` let's configure it to american english
```suggestion
linters-settings:
  misspell:
    locale: US
linters:
```